site/learn/expert-advice/authentication/webauthn-explained.md

```diff
@@ -0,0 +1,478 @@
+        "authenticatorGetInfo": {
+          "versions": [
+            "FIDO_2_0"
+          ],
+          "aaguid": "08987058cadc4b81b6e130de50dcbe96",
+          "options": {
+            "plat": true,
+            "rk": true,
+            "up": true
+          },
+          "maxCredentialCountInList": 100,
+          "maxCredentialIdLength": 32,
+          "transports": [
+            "internal"
+          ],
+          "algorithms": [
+            {
+              "type": "public-key",
+              "alg": -257
+            }
+          ],
+          "minPINLength": 4,
+          "firmwareVersion": 19042
+        }
+      },
+      "statusReports": [
+        {
+          "status": "FIDO_CERTIFIED_L1",
+          "effectiveDate": "2020-08-05",
+          "certificationDescriptor": "Windows Hello Hardware Authenticator",
+          "certificateNumber": "FIDO20020190418002",
+          "certificationPolicyVersion": "1.3.6",
+          "certificationRequirementsVersion": "1.1.0"
+        }
+      ],
+      "timeOfLastStatusChange": "2020-08-05"
+    }
+```
+
+There is a lot of structured metadata that you can use if desired.
+
+## User Presence vs User Verification
+
+Authenticators can be also used as a second factor of authentication, when paired with a username and password or other factor. An authentication ceremony gives you assurances about the user having access to the authenticator. If you are using an authenticator in this way, you want to distinguish between user presence versus user verification. 
+
+User presence is when someone is there behind the authenticator who gave permission, but you don't know who. All it proves is that you have access to the authenticator. An example of this is when you press a button on a Yubikey.
+
+User verification, on the other hand, is when the user provides proof that they are who they say they are. The authenticator could require a pin, known only to the user. It could be a Face ID or Touch ID challenge. In any case, verification allows you to tie a specific user to the authentication ceremony.
+
+Of the two types, user verification is more prevalent with the platform authenticators than cross platform authenticators. And user presence is more useful as a second factor. When not used for MFA, you will typically want user verification.
+
+The initiating server specifies whether to use user verification. The three choices for user verification are preferred, required, and discouraged:
+
+* Preferred means that user verification is preferred, but not required. 
+* Required means that user verification is required and the authentication will fail if it cannot be performed. 
+* Discouraged means that user verification is not preferred and should not be used if possible. 
+
+## Resident Keys
```

Comment:
"Resident key" was [re-termed](https://www.w3.org/TR/webauthn-2/#client-side-discoverable-credential) "client-side discoverable credential" or "discoverable credential" in Level 2 of the spec.

There are lots of fields that still refer to it as a "resident key" for backwards compatibility.